### PR TITLE
feat(trip):Reject Trip Request

### PR DESCRIPTION
**Pull request by alaintresor**
#### What does this PR do?
Reject Trip request by manager
#### How should this be manually tested?

- clone
- npm install
- npm start

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201685
#### Screenshot
Screenshot (52)

### CODE REVIEW

File: src/redux/requests/selectedRequestSlice.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const selectedRequestSlice = createSlice({
+  name: "selectedRequest",
+  initialState: {
+    data: "",
+  },
+  reducers: {
+    selectRequest: (state, action) => {
+      state.data = action.payload;
+      console.log("hhhhhhhhhhhhhhhhhhhhhhhhh");
```
Comment (joshwambere) [inline src/redux/requests/selectedRequestSlice.js]: delete this

**Review by joshwambere — CHANGES_REQUESTED** [review]
Screenshot from 2022-08-25 15-06-01
filll the table in the remaining spaces,

Screenshot from 2022-08-25 15-07-02

this modal does not resember the other modal we have in the system tweak it a little bit

File: src/redux/requests/selectedRequestSlice.js (hunk 1 above)
Comment (joshwambere) [reply src/redux/requests/selectedRequestSlice.js]: also when rejecting or accepting please add some kind of loader to show the user something is happening

**Review by joshwambere — DISMISSED** [review]
Rebase, and find reason why trip request are not getting loaded on initial mounting of trip request component

**Review by Patricknshimiyimana — APPROVED** [review]

**Review by mwisemarierose — APPROVED** [review]

**Review by Placide123 — APPROVED** [review]

**Pull request merged**